tests/conftest.py

```diff
@@ -1174,6 +1175,7 @@ def setup_for_snippet(
         :param strict_deps_check: True iff the returned project should have strict dependency checking enabled.
         :param use_pip_config_file: True iff the pip config file should be used and no source is required for v2 to work
                                     False if a package source is needed for v2 modules to work
+        :param main_file: Relative path from the project root to the .cf file to use as main entry point.
```

Comment:
This can also be an absolute path, no?
```suggestion
        :param main_file: Path to the .cf file to use as main entry point. A relative or an absolute path can be provided. If a relative path is used, it's interpreted relative to the root of the project directory.
```

Reply:
Hmm I don't think so. Unless I'm missing a usage it's currently used as a relative path:
- https://github.com/inmanta/inmanta-core/blob/71c9e88c6f9621dd31f3cd50d6b8f627d232febf/tests/conftest.py#L1304
- https://github.com/inmanta/inmanta-core/blob/71c9e88c6f9621dd31f3cd50d6b8f627d232febf/tests/conftest.py#L1207

Reply:
I don't see what would fail. It's possible to join a two absolute paths together:

```
>>> import os.path
>>> 
>>> a = "/tmp/test"
>>> b = "/root/test2"
>>> os.path.join(a,b)
'/root/test2'
>>> 
>>> a = "/tmp/test"
>>> b = "root/test2"
>>> os.path.join(a,b)
'/tmp/test/root/test2'
>>> 

```

Reply:
Wow I didn't know that!

> If a path component represents an absolute path, then all previous components joined are discarded, and joining continues from the absolute path component.